src/tests/Microsoft.VisualStudio.Composition.Tests/RuntimeCompositionTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Microsoft.VisualStudio.Composition.Tests
+{
+    using System;
+    using Xunit;
+
+    public class RuntimeCompositionTests
+    {
+        [Fact]
+        public void TestEmptyCatalogTest()
+        {
+            var configuration = CompositionConfiguration.Create(TestUtilities.EmptyCatalog);
+            var composition = RuntimeComposition.CreateRuntimeComposition(configuration);
+            var exports = composition.GetExports(typeof(IDisposable).ToString());
```

Comment:
Using `.ToString()` seems odd here, as passing in an explicit contract name doesn't resemble the common use case in VS-MEF. Why not use `GetExports<IDisposable>()`?
But I think it serves the purpose either way. I just tend to prefer unit tests that look like the most common use case possible while still testing the intended thing. That way, if it fails, we can readily see how impactful the failure might be.